src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/BamboraConfiguration.java

```diff
@@ -12,28 +10,25 @@
 @RestController
 public class BamboraConfiguration {
 
-	@Value("${dps.crc.endpoint.approved}")
-	private String approved;
-	@Value("${dps.crc.endpoint.declined}")
-	private String declined;
-	@Value("${dps.crc.endpoint.error}")
-	private String error;
+    @Value("${dps.crc.endpoint.approved}")
+    private String approved;
+    @Value("${dps.crc.endpoint.declined}")
+    private String declined;
+    @Value("${dps.crc.endpoint.error}")
+    private String error;
 
-	@RequestMapping(value = "/bamboraconfiguration", method = RequestMethod.GET)
-	public BeanstreamEndpointResponse singlepaymenturl() {
-		try {
-			// Make sure autowired params are not null (May be missing from configMap)
-			if ( StringUtils.isEmpty(approved) || StringUtils.isEmpty(declined) || StringUtils.isEmpty(error))
-				throw new PaymentServiceException(PaymentServiceConstants.PAYMENT_SERVICE_ERR_MISSING_CONFIG_PARAMS);
-			return new BeanstreamEndpointResponse(approved, declined, error,
-					PaymentServiceConstants.PAYMENT_SERVICE_RESP_MSG_OK,
-					PaymentServiceConstants.PAYMENT_SERVICE_SUCCESS_CD);
-		}
-		catch (PaymentServiceException e)
-		{
-			e.printStackTrace();
-			return new BeanstreamEndpointResponse("","","",PaymentServiceConstants.PAYMENT_SERVICE_RESP_MSG_FAIL,
+    @RequestMapping(value = "/bamboraconfiguration", method = RequestMethod.GET)
+    public BeanstreamEndpointResponse singlepaymenturl() {
+		// TODO - Complete logging once available.
+        // Make sure autowired params are not null (May be missing from configMap)
+        if (StringUtils.isEmpty(approved) || StringUtils.isEmpty(declined) || StringUtils.isEmpty(error)) {
```

Comment:
you can also remove the braces from this if statement if the else is being removed, making this a clean one liner.